dev-setup.sh

```diff
@@ -1,18 +1,73 @@
-sfdx force:org:create -v devhub -a sat-dev -f config/project-scratch-def.json -d 7 -s;
-sfdx force:source:push -u sat-dev;
-sfdx force:user:permset:assign -u sat-dev -n SA_Administrator;
-sfdx force:apex:execute -u sat-dev -f scripts/apex/set-current-user.apex;
-sfdx force:data:bulk:upsert -u sat-dev -s Drug__c -f data/drugs.csv -i Drug_Code__c -w 5;
-sfdx force:data:bulk:upsert -u sat-dev -s Account -f data/accounts.csv -i Id -w 5;
-sfdx force:data:bulk:upsert -u sat-dev -s Case -f data/cases.csv -i Id -w 5;
-sfdx force:data:tree:import -p data/comm-plan.json;
-sfdx force:apex:execute -u sat-dev -f scripts/apex/add-comm-users.apex;
-sfdx force:apex:execute -u sat-dev -f scripts/apex/assign-cases-to-ecs.apex;
-sfdx force:apex:execute -u sat-dev -f scripts/apex/assign-cases-to-queue.apex;
-sfdx force:apex:execute -u sat-dev -f scripts/apex/add-form-questions.apex;
-sfdx force:data:bulk:upsert -u sat-dev -s Adjudication_Criteria__c -f data/adjudicationcriteria.csv -i Id -w 5;
-sfdx force:data:bulk:upsert -u sat-dev -s Diagnosis__c -f data/diagnosis.csv -i Id -w 5;
-sfdx force:data:bulk:upsert -u sat-dev -s Drug_Template__c -f data/drugtemplates.csv -i Id -w 5;
-sfdx force:data:bulk:upsert -u sat-dev -s Product_Health_Category__c -f data/producthealthcategories.csv -i Id -w 5;
-sfdx force:apex:execute -u sat-dev -f scripts/apex/assign-drug-default-queues.apex;
-sfdx force:org:open -u sat-dev;
\ No newline at end of file
+#!/bin/bash
+
+help() {
+   echo ""
+   echo "Usage: $0 -a <string>"
+   echo -e "\t-a Set an alias for scratch org. Default 'sat-dev'."
+   echo -e "\t-v Print each command and its output. Default 'true'"
+   exit 1 # Exit script after printing help
+}
+
+verbose=false;
+alias='sat-dev';
+
+while getopts a:v opt 
+do
+   case "$opt" in
+   (a) alias="$OPTARG" ;;
+   (v) verbose=true ;;
+   (?) help ;;
+   esac
+done
+
+dx() { 
+    if [ "$verbose" = true ]
+    then
+      # Print command and run.
+      echo "\$ sfdx ${@/eval/}" ; sfdx "$@" ;
+      status=$?
+    else
+      # Suppress sfdx's output, and read the status property of the resulting JSON output instead. 
+      result=$(sfdx "$@" --json 2> /dev/null);
+      status=$(jq .status <<< $result)
```

Comment:
@marklise I'm using `jq` to parse the JSON output from `sfdx` and read the value of the `status` property. Any ideas without needing to install the library? It's used in line 40 as well.

Reply:
change those two lines to the following:

```
status=`echo $result | sed -e 's/^.*"status"[ ]*:[ ]*//' -e 's/\,.*//'`
echo $status
```